sdk/helper/scaleutils/node_drain.go

```diff
@@ -19,6 +19,9 @@ import (
 const (
 	defaultNodeDrainDeadline    = 15 * time.Minute
 	defaultNodeIgnoreSystemJobs = false
+
+	NODE_DRAINED_META_KEY   = "drained_by"
+	NODE_DRAINED_META_VALUE = "nomad-autoscaler"
```

Comment:
These should use camel case instead of snake case and I don't believe they need to be exported.
```suggestion
	nodeDrainedMetaKey   = "drained_by"
	nodeDrainedMetaValue = "nomad-autoscaler"
```